about/about-network-policy.md

```diff
@@ -0,0 +1,223 @@
+---
+title: About Network Policy
+description: Learn about networking!
+---
+
+> <span class="glyphicon glyphicon-info-sign"></span> This guide provides optional background education, including
+> education that is not specific to {{site.prodname}}.
+{: .alert .alert-info}
+
+### What is network policy?
+
+Network policy is the primary tool for securing a Kubernetes network. It allows you to easily restrict the network traffic in
+your cluster so the only traffic that is allowed to flow is the traffic you are wanting to flow.
```

Comment:
It allows you to easily restrict the network traffic in your cluster so only traffic that you want to flow is allowed.